apps/api/src/domain/models/bibliographic-reference/types/Creator.ts

```diff
@@ -0,0 +1,25 @@
+import { DTO } from '../../../../types/DTO';
```

Comment:
This is a class, not a type. It should be in `.../biblliographic-reference/entities/creator.entity.ts`

Reply:
Also, this class should be called `ZoteroCreator` or `BibliographicReferenceWorkCreator` or something like that, not just `Creator`.

Reply:
Fixed this in JournalArticle branch PR (#75)